### PR TITLE
change landing constants

### LINKED ISSUE #6948: Rename landing constants
See: https://github.com/mozilla/addons-frontend/pull/6932#discussion_r234034218

---

We should rename the landing constants so that constants and action creators have similar names.

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #6948

### CODE REVIEW

File: src/amo/reducers/landing.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ type GetLandingParams = {|
 |};
 
 export type GetLandingAction = {|
-  type: typeof LANDING_GET,
+  type: typeof FETCH_LANDING,
```
Comment (willdurand) [inline src/amo/reducers/landing.js]: I'd prefer `GET_LANDING`, because that's the name of the action.

**Comment by willdurand** [comment]
@championshuttler can you please rebase?

**Comment by championshuttler** [comment]
> @championshuttler can you please rebase?

I think that's why tests were failing yesterday.

**Comment by willdurand** [comment]
Awesome, thanks @championshuttler!

**Pull request merged**